src/features/layers_in_area/atoms/areaLayersDetailsResource/areaLayersDetailsParamsAtom.ts

```diff
@@ -69,7 +69,10 @@ export const areaLayersDetailsParamsAtom = createAtom(
         const eventFeed = getUnlistedState(currentEventFeedAtom);
         if (eventFeed) newState.eventFeed = eventFeed.id;
       } else {
-        throw Error('Current geometry not from event, event related layer was selected');
+        // TODO: investigate why we needed to throw Error here before
```

Comment:
because it's not expected to retrieve layer-related layer (for example Event Shape) when event id is not presented

Reply:
this one breaks overlay selection after you re
<img width="1792" alt="Screenshot 2023-01-09 at 12 47 09" src="https://user-images.githubusercontent.com/20676525/211280425-2973c8ae-8f90-451e-ac5b-75370ae2bbd6.png">
move selected geometry

Reply:
that's the bug it was hiding then 😄

Reply:
so what to do with this line in PR? revert and fix in separate bug?

Reply:
@IlyaIzr you can throw errors only if you somewere catch them. Never throw just for